### DISCUSSION TITLE
Proper way to getting rid of package "deployphp/recipes"

### DISCUSSION THREAD

**Discussion by dkoether** [deployphp/deployer#2467] [Help needed] [answered]
Hello everyone,

I've used the "deployphp/recipes" which are moved to https://github.com/deployphp/deployer/tree/master/contrib and the repository is abandoned. I am using version 6.8.0 right now but the recipes are currently only on branch 'master'. 

My question is: What is the correct way to migrate from deployphp/recipes but not using any beta version of deployer? Do I have to use the abandoned package and receive the associated hint whenever I am using composer or will there be a new version released soon?

Thanks in advance and keep up the good work! 👍🏻

**Comment by calebanthony** [comment, marked as answer]
In [this](https://github.com/deployphp/deployer/issues/2122) thread the dev says:

> Old versions are still tagged and working. Abandoned is added notify contributors what recipes should be updates in master repo instead.

**Reply by dkoether** [reply]
Thank you Caleb! Until deployer in Version 7.0 is released there seems to be no other way than using the abandoned repository deployer/recipes.

**Reply by calebanthony** [reply]
Yeah it seems like it's only abandoned in that it won't be updated any more and developers shouldn't contribute to it or open issues. It won't be deprecated until V7 is stable release.

**Reply by dkoether** [reply]
Nonetheless this way is inconvenient because the recipes are moved but no stable update instructions are given nor a major release for 6.X was released to make this done completely.

So all we can do is waiting for version 7.0, but I am fine with that.
